RetrospectiveExtension.Frontend/package.json

```diff
@@ -50,6 +52,7 @@
     "@types/uuid": "8.3.1",
     "@typescript-eslint/eslint-plugin": "4.28.3",
     "@typescript-eslint/parser": "4.28.3",
+    "copy-webpack-plugin": "^6.4.1",
```

Comment:
We may not need this package. It's my bad leaving it in the commit I shared to the team.